### ISSUE TITLE
Change from gamma=0.4 to default in Example Docu

### ISSUE THREAD

**Issue by LutzFassl** [Stable-Baselines-Team/stable-baselines3-contrib#77]
Dear sb3-contrib creators, thanks for this awesome repo.

I have just one small suggestion for the Documentation which might have a big impact for the user:
https://github.com/Stable-Baselines-Team/stable-baselines3-contrib/blob/master/docs/modules/ppo_mask.rst

In this Example of MaskablePPO the gamma is set to **0.4**
Pasting this snippet to own code and overlooking this drastic hyperparameter setting can easily lead to non-learning models.
I suggest to set gamma to default 0.99 or remove the argument from this example. BR

**Comment by Miffyli** [comment]
Hey! Thanks for the kind words and the suggestion :). I assume 0.4 was chosen because it is good for that specific case (can speed up training if env is simple), but I agree, it should be closer to a good default (i.e., 0.99). At the very least, there should be a warning about this as it is a very different value from what you usually use. We would happily accept a PR on this, unless @araffin  has other thoughts on this :)

**Comment by araffin** [comment]
> At the very least, there should be a warning about this as it is a very different value from what you usually use

yes, I would be for a comment just above it that explains why we chose a non-conventional value in that example.
